srcpkgs/libgcrypt/patches/hwf-x86-fix-wrong-operand-type.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/src/hwf-x86.c b/src/hwf-x86.c
```

Comment:
Can you leave a link for the origin of the patch? I guess it's the one that fixes the i686 failing test?

Reply:
https://github.com/gpg/libgcrypt/commit/ec87511d9cd2dc31434e939b6351d74a38d4ceaa
Yeah it seems that it fixes the i686 tests. Adding the patch with some more info instead of just the diff now.